doc/source/zuul.rst

```diff
@@ -65,5 +66,354 @@ each commit is correctly tested.
 Zuul's current status may be viewed at
 `<https://zuul.opendev.org/>`_.
 
+Software Architecture
+=====================
+
+Please refer to `zuul`_ documentation for detailed explanation on how Zuul is designed.
+
+.. raw:: html
+
+   <object data="_static/zuul.svg" type="image/svg+xml"></object>
+
+Security Design
+---------------
+
+Security Architecture
+~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+   <object data="_static/zuul_sec.svg" type="image/svg+xml"></object>
+
+Separation
+~~~~~~~~~~
+
+Zuul consists of the following major components:
```

Comment:
I know that these components are described in more details in upstream zuul docs, but shouldn't we add at least a one liner what each component is used for ?